src/includes/getting-started-primer/unity.mdx

```diff
@@ -2,6 +2,8 @@ Our Unity SDK builds on top of the [.NET SDK](/platforms/dotnet/) and extends it
 
 **Additional Features:**
 
-- Unity's `Debug.Log` and `Debug.LogWarning` get captured with [breadcrumbs](/product/issues/issue-details/breadcrumbs/)
-- Scene loading and unloading get captured with [breadcrumbs](/product/issues/issue-details/breadcrumbs/)
+- [Offline caching](/platforms/unity/configuration/options/#cache-directory-path) when a device is unable to connect; we send a report once we receive another event
```

Comment:
```suggestion
- [Offline caching](/platforms/unity/configuration/options/#cache-directory-path) stores data to disk in case the device is not online
```